src/Middleware/HttpOverrides/src/HttpMethodOverrideExtensions.cs

```diff
@@ -7,6 +7,9 @@
 
 namespace Microsoft.AspNetCore.Builder
 {
+    /// <summary>
+    /// Extension methods for enabling <see cref="HttpMethodOverrideMiddleware"/>.
+    /// </summary>
     public static class HttpMethodOverrideExtensions
```

Comment:
Can we add a snippet to the description of the `Use*` methods that say when you would use this feature? Maybe something like 

> This middleware is used when a client wants to express arbitrary HTTP methods but is limited to sending GET or POST methods. By default, the `X-HTTP-Method-Override` request header is used to specify the HTTP method being tunneled.

Reply:
Updated